axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConnectionManager.java

```diff
@@ -709,6 +716,12 @@ public Builder scheduler(ScheduledExecutorService scheduler) {
             return this;
         }
 
+        public Builder axonFrameworkVersionResolver(Supplier<String> axonFrameworkVersionResolver) {
+            assertNonNull(axonFrameworkVersionResolver, "AxonConnectorProperties may not be null");
```

Comment:
The message added to the `AxonConfigurationException` is incorrect, as it states `AxonConnectorProperties` whilst this is a `Supplier` of `String` verification.